lib/actions/copy.js

```diff
@@ -63,5 +63,10 @@ exports._copySingle = function (from, to, options) {
     contents = applyProcessingFunc(options.process, file.contents, file.path);
   }
 
+  for (var key in context) {
+    to = to.replace('<%=' + key + '%>', context[key]);
```

Comment:
It'd be better to use the template engine to compile the path string the same way we compile the files as templates. Using replace is too basic and might creates inconsistent behavior between the path and the file templates themselves.

Reply:
I agree. Modified implementation to:
to = ejs.render(to, context, tplSettings);

Also added if condition to render only if context is not null.